zombies.js

```diff
@@ -316,7 +476,12 @@
  * -----------------------------
  */
 
-
+class ExplodingZombie extends Zombie{
+  constructor(health, strength, speed){
+    super(health, strength, speed);
+    this.speed = speed;
```

Comment:
speed is part of the super constructor, so I does not need to be set on the child